sklearn/utils/_testing.py

```diff
@@ -758,30 +758,34 @@ def assert_run_python_script(source_code, timeout=60):
         os.unlink(source_file)
 
 
-def _convert_container(container, constructor_name, columns_name=None):
+def _convert_container(container, constructor_name, columns_name=None,
```

Comment:
Could you add a new test in `test_testing` specifically to check that `dtype` works as expected.

Reply:
I added this test such that we can merge for the release